src/vcf/odb_report.cpp

```diff
@@ -73,6 +73,11 @@ namespace ebi
         if (transaction.has_current()) {
             transaction.commit();
         }
+
+        {
+            odb::core::connection_ptr c{db->connection()};
```

Comment:
We are doing this each million variants, a quick profile shows that the whole flush method is a 0.03% of the time (including the Error write commiting), and I'm not sure if the connection is using transactions underneath or not. If it is using them, we can not put it at the beginning, and if it's not, it's already fast.